### ISSUE TITLE
Enhancement of public serverlist in the client

### ISSUE THREAD

**Issue by hunter688** [minetest/minetest#2568]
When someone looking for server, would like search using some criteria:
- num of players online
- ping/latency of server
- Nation (can be used geoip) of server
- others filters (pvp,creative)
  .....

Will be fine add some filtering/sorting features to serverlist.

**Comment by SmallJoker** [comment]
This might be a theme when there are over 50 servers online at once.
Currently, it's easy to find a green/white colored server with the desired icons.

**Comment by paramat** [comment]
Any more comments?
Is anyone interested in coding this sometime? if not closing in 1 month.

**Comment by SmallJoker** [comment]
@paramat We never added PvP, player count and ping filters to the multiplayer tab. Not implemented feature request -> reopening.

**Comment by paramat** [comment]
Yes i know filters were not added. Ok, i interpreted your comment as being a disapproval. Now we have more servers this is more useful.

**Comment by paramat** [comment]
@SmallJoker so do you support this request?
Any other core dev support?

**Comment by SmallJoker** [comment]
@paramat The idea itself is surely interesting and there's also space (extending downwards) for filter options available in the Multiplayer tab.

**Comment by nerzhul** [comment]
:+1:
